File: branches/Medusa/Source/Fantasy.Jobs/When.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;

namespace Fantasy.Jobs
{
    [Instruction]
    [XSerializable("when", NamespaceUri=Consts.XNamespaceURI )] 
    internal class When : Sequence
    {
        [XAttribute("condition")]
        public string Condition = null;

        public override void Execute()
        {
            this.ExecuteSequence(); 
        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/When.java
package Fantasy.Jobs;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[Instruction, XSerializable("when", NamespaceUri=Consts.XNamespaceURI)]
public class When extends Sequence
{
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("condition")]
	public String Condition = null;

	@Override
	public void Execute()
	{
		this.ExecuteSequence();
	}
}